unit-tests/src/test/java/org/eclipse/collections/impl/multimap/AbstractMutableMultimapTestCase.java

```diff
@@ -132,6 +135,33 @@ public void testPutAll()
     }
 
     @Test
+    public void testPutAllPairs()
+    {
+        MutableMultimap<Integer, String> multimap1 = this.newMultimapWithKeysValues(1, "One", 2, "2");
+        MutableList<Pair<Integer, String>> pairs1 = Lists.mutable.of(Tuples.pair(1, "One"), Tuples.pair(2, "Two"), Tuples.pair(3, "Three"));
+        Assert.assertTrue(multimap1.putAllPairs(pairs1));
+        MutableMultimap<Integer, String> expected1 = this.newMultimapWithKeysValues(1, "One", 1, "One", 2, "2", 2, "Two");
+        expected1.put(3, "Three");
+        Assert.assertEquals(expected1, multimap1);
```

Comment:
The first three lines make a good test of `putAllPairs()`. I find lines 4 and 5 confusing, which mutate the expected Multimap. Is it just because the factories only have overloads going up to 4 keys? If so, I'd prefer to work around that limitation in a more consistent way.